### PR TITLE
Utils for constructing v2 mutations

### PR DESCRIPTION
**Pull request by willmarks**
## Description

Some helpers when constructing v2 mutations.

## Type of change
- [ ] Bug fix
- [X] New feature
- [ ] Breaking change

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "tsc -p src",
     "clean": "rm -rf lib node_modules out",
-    "fix": "npm run lint -- --fix && npm run pretty",
+    "fix": "npm run lint -- --fix",
```
Comment (willmarks) [inline package.json:18]: `npm run pretty` messes stuff up so that it no longer passes lint.

**Comment by willmarks** [comment]
Let me know if you think these would be helpful to include for people trying to build v2 mutations.

**Review by ted-faros — COMMENTED** [review]
@willmarks can you provide some sort of example usage in the form of docs or test?

**Review by ted-faros — APPROVED** [review]
thanks for the usage info

File: README.md [hunk 2]
```diff
@@ -30,6 +30,77 @@ const query = `{
 const data = await client.gql('default', query);
 ```
 
+## GraphQL Query Builder
+
+The QueryBuilder class is a utility to help construct graphQL mutations from Faros models.
```
Comment (tovbinm) [inline README.md:35]: `graphQL` -> `GraphQL`

File: README.md [hunk 3]
```diff
@@ -30,6 +30,77 @@ const query = `{
...
+    mutations.push(
+      qb.upsert(application),
+      qb.upsert(organization),
+      qb.upsert(pipeline),
+      qb.upsert(build),
+      qb.upsert(deployment)
+    );
+
+    // Send your mutations to Faros!
+    client.sendMutations(mutations);
```
Comment (tovbinm) [inline README.md:101]: is this sync or async call?

Comment (willmarks) [reply README.md:101]: async. Will add an await

**Pull request merged**